tests/scripts/amsr2.config_based_no_compare.sh

```diff
@@ -15,8 +15,12 @@
 run_procflow ${GEOIPS_TESTDATA_DIR}/test_data_amsr2/data/AMSR2-MBT_v2r2_GW1_s202005180620480_e202005180759470_c202005180937100.nc \
   --procflow config_based \
   --output_config ${GEOIPS_PACKAGES_DIR}/geoips/tests/yaml_configs/amsr2_test_no_compare.yaml \
+  --reader_kwargs \
```

Comment:
Again, put these in existing test scripts just so I could make sure it works.  Probably a good time to set up some unit tests instead (though we would still need some sort of integration test if it is a new command line argument)

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/334

@jsolbrig Please resolve and approve this PR when ready.  We will add unit tests at a later date.